src/System.Net.Requests/tests/HttpWebRequestHeaderTest.cs

```diff
@@ -102,5 +101,62 @@ public void HttpWebRequest_EndGetRequestStreamContext_Null(Uri remoteServer)
                 Assert.Equal(null, context);
             }
         }
+
+        [OuterLoop]
+        [Theory, MemberData(nameof(EchoServers))]
+        public void HttpWebRequest_Connection_Validate(Uri remoteServer)
+        {
+            HttpWebRequest request = WebRequest.CreateHttp(remoteServer);
+            Assert.Throws<ArgumentException>(() => { request.Connection = "Close;keep-alive"; });
+        }
+
+        [OuterLoop]
+        [Theory, MemberData(nameof(EchoServers))]
+        public void HttpWebRequest_DefaultCachePolicy_Validate(Uri remoteServer)
+        {
+            Assert.Equal(Cache.RequestCacheLevel.BypassCache, HttpWebRequest.DefaultCachePolicy.Level);
+        }
+
+        [OuterLoop]
+        [Theory, MemberData(nameof(EchoServers))]
+        public void HttpWebRequest_Proxy_Validate(Uri remoteServer)
```

Comment:
Nit: In general, the names should be enough to understand what's expected: 
1. What module/class is it testing
2. What is the scenario
3. Is it expected to pass fail, etc

In this case a good name would be `HttpWebRequest_ProxySetAfterGetResponse_Fails`.